pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -1158,6 +1158,9 @@ protected CompletableFuture<Void> internalSplitNamespaceBundleAsync(String bundl
                 .thenCompose(__ -> getNamespacePoliciesAsync(namespaceName))
                 .thenCompose(policies->{
                     String bundleRange = getBundleRange(bundleName);
+                    if (bundleRange == null) {
+                        throw new RestException(Status.PRECONDITION_FAILED, "Failed to find a bundleRange");
```

Comment:
It should be 404, not 412?
And the error message should be `Bundle range xxx not found`